src/comp.py

```diff
@@ -23,49 +23,49 @@ def __repr__(self):
 # whose name starts with 'D':
 
 print("Starts with D:")
-r = []  # TODO
+r = [i.name for i in humans if i.name[0]=='D']  # TODO
 print(r)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 
 print("Ends with e:")
-r = []  # TODO
+r = [i.name for i in humans if i.name[-1] == 'e']  # TODO
 print(r)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with any letter between 'C' and 'G' inclusive.
 
 print("Starts between C and G, inclusive:")
-r = []  # TODO
+r = [i.name for i in humans if ord(i.name[0]) >= 67 and ord(i.name[0]) <= 71]  # TODO
 print(r)
 
 # Write a list comprehension that creates a list of all the ages plus 10.
 print("Ages plus 10:")
-r = []  # TODO
+r = [i.age for i in humans if i.age > 10]  # TODO
```

Comment:
Does _list of all ages plus 10_ mean _all ages greater than 10_ or _all ages plus 10 added to each_? Confusing instructions...